plugin/pyproject.toml

```
@@ -0,0 +1,26 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "bgutil-ytdlp-pot-provider"
+version = "0.2.1"
+readme = {file = "README.md", content-type = "text/markdown"}
+requires-python = ">=3.8"
+keywords = ["yt-dlp", "yt-dlp-plugin"]
```

```suggestion
keywords = ["yt-dlp", "yt-dlp-plugin", "yt-dlp-plugins-get-pot"]
```
maybe add a keyword?